.github/workflows/ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     name: Test (Library)
     strategy:
       matrix:
-        node-version: [14.x, 15.x, 16.x, 17.x]
+        node-version: [12, x, 14.x, 15.x, 16.x, 17.x]
```

Comment:
I think you made a small mistake here

```suggestion
        node-version: [12.x, 14.x, 15.x, 16.x, 17.x]
```

Reply:
Good catch - I did make a mistake there. Thanks!